src/typechecker/FStar.TypeChecker.PatternUtils.fst

```diff
@@ -115,13 +115,13 @@ let rec elaborate_pat env p = //Adds missing implicit patterns to constructor pa
                 (p, S.is_bqual_implicit imp)::aux formals' pats'
             end
         in
-        {p with v=Pat_cons(fv, aux f pats)}
+        {p with v=Pat_cons(fv, us_opt, aux f pats)}
     | _ -> p
 
 exception Raw_pat_cannot_be_translated
 let raw_pat_as_exp (env:Env.env) (p:pat)
-  : option term 
-  = let rec aux p = 
+  : option (term * list bv)
```

Comment:
What's up with this change of returning `list bv`? The only caller seems to be `Tactics.Hooks`, where is this is being ignored?